.github/labels.yml

```diff
@@ -0,0 +1,277 @@
+- color: fbca04
+  description: ""
+  name: status:to be verified
+- color: f7c6c7
+  description: ""
+  name: status:under discussion
+- color: 8FFCDE
+  name: "status: potential performance implications"
+  description: ""
+
+# ##### area/misc labels ##### #
+- color: FFFF5F
+  description: ""
+  name: external dependency
+- color: 2C5B58
+  description: ""
+  name: 'infrastructure: ci/cd'
+- color: 5F5F5F
+  description: ""
+  name: maintenance
+- color: d7e102
+  description: ""
+  name: tests
+
+# ##### area/code_location labels ##### #
+- color: 0075ca # blue (standard github)
+  description: Improvements or additions to documentation
+  name: documentation
+- color: 0075ca # blue
+  description: ""
+  name: API
+- color: 0075ca # blue
+  description: ""
+  name: API/v2
+- color: 0075ca # blue
+  description: ""
+  name: CLI
+- color: 0075ca # blue
+  description: ""
+  name: component:st2auth
+- color: 0075ca # blue
+  description: ""
+  name: component:st2client
+- color: 0075ca # blue
+  description: ""
+  name: component:st2web
+- color: 0075ca # blue
+  description: ""
+  name: migrations
+- color: 0075ca # blue
+  description: ""
+  name: runners
+- color: 0075ca # blue
+  description: ""
+  name: 'service: action runner'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: api'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: auth api'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: rules engine'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: scheduler'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: workflow engine'
+
+# ##### area/python labels ##### #
+- color: 0075ca # blue
+  description: ""
+  name: python3
+- color: 0075ca # blue
+  description: ""
+  name: unicode
+
+# ##### area/pack_resources labels ##### #
+- color: 003459 # blue
+  description: ""
+  name: action aliases
+- color: 003459 # blue
+  description: ""
+  name: chatops
+- color: 003459 # blue
+  description: ""
+  name: inquiries
+- color: 003459 # blue
+  description: ""
+  name: output schema
+- color: 003459 # blue
+  description: ""
+  name: packs
+- color: 003459 # blue
+  description: ""
+  name: policies
+- color: 003459 # blue
+  description: ""
+  name: 'workflows: action chain'
+- color: 003459 # blue
+  description: ""
+  name: 'workflows: orquesta'
+
+# ##### area/external_services labels ##### #
+- color: 8336ff # blue
+  description: ""
+  name: database
+- color: 8336ff # blue
+  description: ""
+  name: mongodb
+- color: 8336ff # blue
+  description: ""
+  name: nginx
+- color: 8336ff # blue
+  description: ""
+  name: rabbitmq
+
+# ##### area/logging_and_performance labels ##### #
+- color: 8336ff # purple
+  description: ""
+  name: logging
+- color: 8336ff # purple
+  description: ""
+  name: metrics
+- color: 8336ff # purple
+  description: ""
+  name: performance
+
+# ##### area/install labels ##### #
+- color: 50209c # purple
+  description: ""
+  name: deployment
```

Comment:
I think we can use this color of purple for `area/install` labels. Here is how `#50209c` looks in dark and light mode:
![image](https://user-images.githubusercontent.com/1558590/144345640-6caf05be-d07c-4a4e-8113-ef09fba95f61.png)